build/buildPackage.sh

```diff
@@ -127,6 +126,15 @@ manifest="
 
 "
 
+if [ "$#" -lt "1" ]; then
```

Comment:
Right I'll check for renderman artefacts and only package renderman if it's required? Also  should we do the same for Arnold? 

How do you feel replacing each of the shell scripts with a python script?